modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/helper/CPContentHelperImpl.java

```diff
@@ -480,6 +482,14 @@ public String getImageURL(FileEntry fileEntry, ThemeDisplay themeDisplay)
 		return cpMedia.getURL();
 	}
 
+	public String getMiniumOrderQuantity(long cpDefinitionId) {
+		CPDefinitionInventory cpDefinitionInventory =
+			_cpDefinitionInventoryLocalService.
+				fetchCPDefinitionInventoryByCPDefinitionId(cpDefinitionId);
+
+		return String.valueOf(cpDefinitionInventory.getMinOrderQuantity());
```

Comment:
why don't you return the number instead of the String?

Reply:
Return number here